trview.ui/Scrollbar.cpp

```diff
@@ -12,7 +12,7 @@ namespace trview
         Scrollbar::Scrollbar(const Point& position, const Size& size, const Colour& background_colour)
             : Window(position, size, background_colour)
         {
-            _blob = add_child(std::make_unique<Window>(Size(size.width, 10), Colour(1.0f, 0.15f, 0.15f, 0.15f)));
+            _blob = add_child(std::make_unique<Window>(Point(1, 1), Size(size.width - 2, 10), Colour(0.2f, 0.2f, 0.2f)));
```

Comment:
I feel like having a .h file with just UI colours may be useful in the future. I see we already have a few standard colours in Colour.h.

Reply:
Made #829